src/Mod/Material/FluidMaterial/Air.FCMat

```diff
@@ -17,7 +17,8 @@ ThermalConductivity = 0.02587 W/m/K
 ; thermal expansion coefficient of ideal gas is 1/temperature
 ThermalExpansionCoefficient = 3.43e-3 1/K
 
-[Electrical]
+[Electromagnetic]
 RelativePermittivity = 1.00059
 ; at 18°C and 50Hz
 ElectricalConductivity = 1e-12 S/m
+RelativePermeability = 1.0
```

Comment:
Do you have a set number of digits of precision you are targeting?

Reply:
No. You will wind in databases the permeability of air to be 1.0000037 or something. This is academical, not value for real-life simulations.
1.0 is sufficient here, especially since air is never perfectly dry.

For copper and water the difference to 1.0 might be of interest. This is however just the preset, who does not like to see a long number, can change it to 1.0.

Reply:
OK, thanks -- I am no expert here, so to me this code looks fine to merge. I can click the button if you like, or you can wait for someone who knows more.

Reply:
Please click since this helps me fiddling around with my ongoing FEM work (addition for electromagnetic simulations)